mslib/mscolab/server.py

```diff
@@ -351,6 +351,20 @@ def get_user():
     return json.dumps({'user': {'id': g.user.id, 'username': g.user.username}})
 
 
+@APP.route('/upload_profile_image', methods=["POST"])
+def upload_profile_image():
+    user_id = request.form['user_id']
+    file = request.files['image']
+    if file:
```

Comment:
and here should be configurable limit over the servers settings config.

I am not sure if I would give a user more than 1MB max size of a file.

Reply:
file.read may need also some security checks.

e.g. is it an image mimtype